shimmingtoolbox/prepare_fieldmap.py

```diff
@@ -3,11 +3,13 @@
 
 import math
 import numpy as np
+from skimage.filters import gaussian
 
 from shimmingtoolbox.unwrap.unwrap_phase import unwrap_phase
 
 
-def prepare_fieldmap(phase, echo_times, affine, unwrapper='prelude', mag=None, mask=None, threshold=None):
+def prepare_fieldmap(phase, echo_times, affine, unwrapper='prelude', mag=None, mask=None, threshold=None,
+                     gaussian_filter=None, sigma=None):
```

Comment:
```suggestion
                     gaussian_filter=False, sigma=None):
```
I would suggest going with a default value of `False` so that it's always a boolean

```suggestion
                     gaussian_filter=None, sigma=1):
```
I think having a default value for sigma would be useful, I have not played with the code to know which default is best.